### PR TITLE
Add hooks to generate API doc for new methods.

### PR DESCRIPTION
**Pull request by kybowm**
@courtneyambrozic , @cshruti87 , 

I was wondering why the recent API additions didn't toggle a new doc build. I had forgotten that we need to add a hook to the doc/src/api.rst. I'm sorry that I missed it during code reviews.

This commit adds the hooks to the doc source. Because the commit includes a change in the doc/ directory, the GitHub action should rebuild the doc.

### CODE REVIEW

**Comment by cshruti87** [comment]
Thanks @kybowm!

File: doc/source/api.rst [hunk 1]
```diff
@@ -52,11 +54,13 @@ Image
     Image
     Image.convert_to_CAS_column
     Image.convert_wide_to_numpy
+    Image.convert_numpyt_to_wide
```
Comment (courtneyambrozic) [inline doc/source/api.rst:57]: I think the "t" should be removed after "numpy" here

**Review by courtneyambrozic — APPROVED** [review]
Thanks, Kyle! 👍

File: doc/source/api.rst (hunk 1 above)
Comment (kybowm) [reply doc/source/api.rst]: Good catch. Thanks! That should be fixed, now.

**Pull request merged**